### DISCUSSION TITLE
Ability to remove/ hide Download as button from the app

### DISCUSSION THREAD

**Discussion by helgakovacs** [marimo-team/marimo#2171] [Ideas]
Hi, 
I would like to request to have the option to remove the _Download as_ button from the top right corner of the app. We are not using this functionality and it might be confusing for the user.

**Comment by mscolnick** [comment]
Hi @helgakovacs - you might be able to remove it using a `custom.css` file: https://docs.marimo.io/guides/theming.html

If not, we can make it configurable. I am not sure where it would sit in our priorities, but we would happily take a contribution as well.
